aspnetcore/fundamentals/openapi/customize-openapi.md

```diff
@@ -42,9 +42,23 @@ Transformers can be registered onto the document by calling the <xref:Microsoft.
 
 ### Execution order for transformers
 
-Transformers execute in first-in first-out order based on registration. In the following snippet, the document transformer has access to the modifications made by the operation transformer:
-
-[!code-csharp[](~/fundamentals/openapi/samples/9.x/WebMinOpenApi/Program.cs?name=snippet_transInOut&highlight=3-9)]
+Transformers are executed as follows:
+
+* Schema transformers are executed when a schema is registered to the document. Schema transformers are executed in the order in which they were added.
+All schemas are added to the document before any operation processing occurs, so all schema transformers are executed before any operation transformers.
+* Operation transformers are executed when an operation is added to the document. Operation transformers are executed in the order in which they were added.
+All operations are added to the document before any document transformers are executed.
+* Document transformers are executed when the document is generated. This is the final pass over the document, and all operations and schemas have been add by this point.
+* When an app is configured to generate multiple OpenAPI documents, each document is generated in the order in which it was registered, and the transformers are executed as documents are generated.
```

Comment:
This statement isn't really true. There are no ordering dependencies between multiple documents. Each document is generated when it is requested via the endpoint.

Reply:
I was thinking of this in the context of build-time document generation ... but maybe that's something we don't need to be explicit about. But we probably should say that transformers for different documents are independent -- don't see each other's results. So perhaps
```suggestion
* When an app is configured to generate multiple OpenAPI documents, transformers are executed for each document independently.
```

Reply:
> was thinking of this in the context of build-time document generation

Yeah, the ordering in the build-time generation isn't explicit. It's just an artifact of the way we get the list of document names in the DI container.

The change you've proposed looks good!